org.eclipse.scout.rt/pom.xml

```diff
@@ -177,6 +177,12 @@
         <groupId>ch.qos.logback</groupId>
         <artifactId>logback-classic</artifactId>
         <version>${logback.version}</version>
+        <exclusions>
```

Comment:
Die Exclusion ist gewollt?

Reply:
Updated to latest version of logback (released last Friday) that does not require the exclusion anymore